src/Http/Routes/Character/View.php

```diff
@@ -123,20 +123,38 @@
         'uses'       => 'IntelController@getTopWalletJournalData',
     ]);
 
+    Route::get('summary/journal/details/{first_party_id}/{getJournalContent}', [
```

Comment:
`getJournalContent` should match the rest of the convention used in SeAT please. aka: `get_journal_content`

Reply:
that's a stupid copy-pasta error 👎 

it's obviously the method from IntelController and should be `second_party_id`